test/emu_mps/test_tdvp.py

```diff
@@ -212,16 +212,16 @@ def test_evolve_single():
         ham_factor=ham_factor,
         dt=dt,
         config=EvolveConfig(
-            exp_tolerance=1e-10,
-            norm_tolerance=1e-10,
-            max_krylov_dim=1000,
+            exp_tolerance=1e-8,
+            norm_tolerance=1e-8,
+            max_krylov_dim=100,
             is_hermitian=False,
-            max_error=1e-10,
-            max_rank=100,  # FIXME: max_error and max_rank are irrelevant for evolve_single
+            max_error=1e-5,
+            max_rank=10,  # FIXME: max_error and max_rank are irrelevant for evolve_single
         ),
     )
 
-    assert torch.allclose(expected, actual, atol=1e-4)
+    assert torch.allclose(expected, actual, atol=1e-8)
```

Comment:
an issue I ran into, is that with the default value of `rtol=1e-5` this would pass even for `atol=1e-20` since the relative error is small enough. I expect this problem to exist in other places in the code too, actually. Perhaps you could do a quick search on all `torch.allclose` instances in the tests?